src/main/java/sirius/db/mixing/properties/LocalDateTimeProperty.java

```diff
@@ -124,11 +125,10 @@ protected Object transformToElastic(Object object) {
 
     @Override
     protected Object transformToMongo(Object object) {
-        if (!(object instanceof LocalDateTime)) {
-            return null;
+        if (object instanceof LocalDateTime localDateTime) {
+            return QueryBuilder.FILTERS.transform(localDateTime == null ? null : localDateTime.withNano(0));
```

Comment:
shouldn't we move this into the transform method ->
MongoFilterFactory.customTransform(Object value)

Becaus we shoud apply "withNano(0)" on all values to be compared against a stored values as otherwise the comparison might fail...